sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - Re-exports `RetryMode` for use when setting the `RetryOptions.mode` field
   in `ServiceBusClientOptions`.
   Resolves [#13166](https://github.com/Azure/azure-sdk-for-js/issues/13166).
+- When receiving messages from sessions using the `ServiceBusSessionReceiver.receiveMessages` method, errors on the AMQP link or session were being handled well, but an error on the AMQP connection like a network disconnect was not being handled at all. This results in the promise returned by the above method never getting fulfilled.
+  This is now fixed in [#13956](https://github.com/Azure/azure-sdk-for-js/pull/13956) to throw `SessionLockLostError` when in `peekLock` mode and to return the messages collected so far when in `receiveAndDelete` mode.
```

Comment:
```suggestion
  This is now fixed in [#13956](https://github.com/Azure/azure-sdk-for-js/pull/13956) to throw `SessionLockLostError`. If using the `receiveMessages` method in `receiveAndDelete` mode, then the messages collected so far are returned to avoid data loss.
```